### PR TITLE
Create new systemd unit files

### PR DESCRIPTION
**Pull request by aludvik**

### CODE REVIEW

**Comment by aludvik** [comment]
Forgot to sign off on a commit

**Comment by aludvik** [comment]
Rebased and added some extra logging.

**Comment by aludvik** [comment]
Fixed isolation id not being in the environment

File: rest_api/packaging/ubuntu/postinst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+user="sawtooth"
+group="sawtooth"
+
+if ! getent passwd sawtooth > /dev/null; then
```
Comment (rbuysse) [inline rest_api/packaging/ubuntu/postinst]: Should the getent tests in all the postinst scripts use the vars set above instead of "sawtooth"?

**Comment by aludvik** [comment]
Fixed postinst files per @rbuysse comment

**Pull request merged**